bibliopixel/drivers/__init__.py

```diff
@@ -0,0 +1 @@
+from . channel_order import ChannelOrder
```

Comment:
Google's Python coding standards, for example, require _all_ symbols that are external to your current module to be qualified (not necessarily full qualified though).  Their claim is that it makes it clearer which variables are local variables and which aren't.

Honestly, I didn't mean to get rid of the __init__.py contents in this review - just prevent them from being a general dumping ground.